pkg/cmd/cli/secrets/new.go

```diff
@@ -18,7 +19,23 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const NewSecretRecommendedCommandName = "new"
+const (
+	NewSecretRecommendedCommandName = "new"
+
+	newLong = `Create a new secret based on a key file or on files within a directory.
```

Comment:
Need a better long description.  First line should be shorter and match the "Short" very closely.